.gitignore

```diff
@@ -27,3 +27,7 @@ bin/
 # fabric
 
 run/
+
+# mac
+
+.DS_Store
```

Comment:
Could you check if there are any other Mac os metadata files that are left behind and add those?